roles/edpm_ssh_known_hosts/tasks/main.yml

```diff
@@ -88,14 +88,17 @@
         path: "{{ ssh_known_hosts_tmp.path }}"
         block: "{{ ssh_known_hosts_lines }}"
         create: true
-        mode: 0644
+        mode: "0644"
 
-    # Workaround https://bugs.launchpad.net/edpm/+bug/1810932
```

Comment:
That webpage doesn't exist. If the bug isn't relevant anymore we should remove the link altogether, if it is we should keep it unchanged, maybe with a note about why it's still with us since OOO.

Reply:
i thought i changed this url, let me push a fix

Reply:
The comment about why this "hack" is needed is still valid. Ansible, when using modules that involves files, create a temporary files and then overwrite the targets. If those files are used by containers, they aren't upgraded.